ci/jenkins/jobs/projects.yaml

```diff
@@ -411,13 +411,13 @@
               #!/bin/bash
               source /home/ubuntu/.bashrc
               sudo ./ci/test-conformance-eks.sh --aws-access-key ${{AWS_ACCESS_KEY}} --aws-secret-key ${{AWS_SECRET_KEY}} \
-                --cluster-name ${{JOB_NAME}}-${{BUILD_NUMBER}} --setup-only
+                --cluster-name ${{JOB_NAME}}-${{BUILD_NUMBER}} --log-mode detail --setup-only
           triggers:
           - timed: H H */2 * *
           publishers:
           - archive:
               allow-empty: true
-              artifacts: 'eks-test.log'
+              artifacts: 'eks-test.log, ci/*.tar.gz'
```

Comment:
just want to make sure that this will be a single tarball per job run (i.e. each run is a new checkout and we don't accumulate artifacts indefinitely under this `ci/` directory)?

Reply:
Yes the work directory will be cleared for each job run